src/main/java/net/sf/jabref/sql/database/MySQL.java

```diff
@@ -0,0 +1,129 @@
+package net.sf.jabref.sql.database;
+
+import net.sf.jabref.model.entry.IdGenerator;
+import net.sf.jabref.sql.DBStrings;
+import net.sf.jabref.sql.Database;
+import net.sf.jabref.sql.SQLUtil;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class MySQL implements Database {
+
+    public static final String DRIVER = "com.mysql.jdbc.Driver";
+
+    private void loadDriver() throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+        Class.forName(DRIVER).newInstance();
+    }
+
+    @Override
+    public Connection connect(String url, String username, String password)
+            throws IllegalAccessException, InstantiationException, ClassNotFoundException, SQLException {
+        loadDriver();
+
+        return DriverManager.getConnection(url, username, password);
+    }
+
+    @Override
+    public String getReadColumnNamesQuery() {
+        return "SHOW columns FROM entries;";
+    }
+
+    @Override
+    public String getCreateTableSQL(Table table) {
+        switch (table) {
+
+        case JABREF_DATABASE:
```

Comment:
Hm, to me, the method is still quite simple despite being long in lines as it comes with almost no complexity. Hence, I would not gain that much by splitting it up. From my perspective, I would have the hassle of maintaining 9 methods, which have to be put in the interface, and I cannot iterate over these 9 methods in the caller (which is currently done as I can iterate over the Table enum to call the method for each table).